.gitignore

```diff
@@ -11,3 +11,5 @@
 /yggd.bash
 /yggdrasil.spec
 /doc/tags.toml
+
+.idea
```

Comment:
This should not be part of a project-level gitignore file. This should be ignored globally in your `$XDG_CONFIG_HOME/git/ignore` file.

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.